browser/js/signup/signup.html

```diff
@@ -1,42 +1,68 @@
-<section id='signup-form'>
-    <h1>Sign Up</h1>
-    <div>
-        <alert type="danger" ng-show="error">
-            {{ error }}
-            <br>
-        </alert>
+<div class="row">
```

Comment:
we should go over things monday so you can walk us through what was actually changed - very hard to look through when everything appears as green when it shouldn't